scripts/mssql-seed.sh

```diff
@@ -1,4 +1,6 @@
 #!/bin/bash
+#set -euo pipefail
+PATH="/opt/mssql-tools/bin:$PATH"
 
 # Environmentals
 # Optional:
```

Comment:
In [this](https://github.com/Inforitnl/mssql-seed-container/pull/1/files#diff-83fcb7286ae1a7a5f886c1e047bd6a6406b9bdbee797c40948bb7c92af33babcL27) for loop you might want to check for files starting with a number, assuming you want all files to be sequentially numbered.

Reply:
I already mentioned this in the README that's why I didn't want to make it a requirement.